cs-minibot-platform-src/src/main/java/simulator/simbot/Dijkstras.java

```diff
@@ -0,0 +1,199 @@
+package simulator.simbot;
+
+import java.util.*;
+
+/**
+ * Created by jimmychen on 4/29/17.
+ */
+public class Dijkstras {
+
+    //public static int[][] path;
+    public static Node targetNode;
+    private static HashSet<Node> settledNodes = new HashSet<Node>();
+    private static HashSet<Node> unsettledNodes = new HashSet<Node>();
+    private static HashMap<Node, Integer> distances;
+    public static Map<Node, Node> predecessors = new HashMap<Node, Node>();
+
+    //Fills maze with 1's
+    public static int[][] initializeMaze(int n, int m) {
+        int[][] maze = new int[n][m];
+        for(int i = 0; i < n; i++) {
+            for(int j = 0; j < m; j++) {
+                maze[i][j] = 1;
+            }
+        }
+        //path = new int[n][m];
+        return maze;
+    }
+
+    //Takes the given maze and returns that maze wrapped in 1's
+    public static int[][] generateWrappedMaze(int[][] maze) {
+        int height = maze.length;
+        int width = maze[0].length;
+        int[][] wrappedMaze = initializeMaze(height+2,width+2);
+        for(int i = 1; i < height +1; i++) {
+            for(int j = 1; j < width + 1; j++) {
+                wrappedMaze[i][j] = maze[i-1][j-1];
+            }
+        }
+        return wrappedMaze;
+    }
+    //Makes the Nodes array but doesn't fill in neighbors
+    //Also sets cost to max int
```

Comment:
it looks like it only sets the cost to max int if it's not occupied? 

try making this spec slightly more specific- also you should use
/**
* 
*/
for javadoc compatible comments